lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -149,18 +150,15 @@ <h3 class="course-title">
 
               % if share_settings:
                 <%
-                  if share_settings.get("CUSTOM_COURSE_URLS", False):
-                    if course_overview.social_sharing_url:
-                      share_url = urllib.quote_plus(course_overview.social_sharing_url)
-                    else:
-                      share_url = ''
-                  else:
-                    share_url = urllib.quote_plus(request.build_absolute_uri(reverse('about_course', args=[unicode(course_overview.id)])))
+                  share_url = urllib.quote_plus(get_link_for_about_page(course_overview))
                   share_window_name = 'shareWindow'
                   share_window_config = 'toolbar=no, location=no, status=no, menubar=no, scrollbars=yes, resizable=yes, width=640, height=480'
                 %>
-                % if share_url and share_settings.get('DASHBOARD_FACEBOOK', False):
-                  <% facebook_url = 'https://www.facebook.com/sharer/sharer.php?u=' + share_url %>
+                % if share_settings.get('DASHBOARD_FACEBOOK', False):
+                  <%
+                      share_text = _("I'm taking {course_name} online with edX.org. Check it out!").format(course_name=course_overview.display_name_with_default)
```

Comment:
@sstack22 we're making this hardcode since this feature won't be enabled on white label sites. also, according to my little knowledge white label sites uses their own templates.